src/SFA.DAS.EAS.Web/FileDownloads/Payments/CsvTransactionFormatter.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using SFA.DAS.EAS.Domain.Models.Transaction;
+using SFA.DAS.EAS.Web.ViewModels;
+
+namespace SFA.DAS.EAS.Web.FileDownloads.Payments
+{
+    public class CsvTransactionFormatter : ITransactionFormatter
+    {
+        public string MimeType => "text/csv";
+
+        public string FileExtension => "csv";
+
+        public DownloadFormatType DownloadFormatType =>
+            DownloadFormatType.Csv;
+
+        public byte[] GetFileData(List<TransactionDownloadLine> transactions)
+        {
+            var builder = new StringBuilder();
+
+            WriteHeaderCsv(builder);
+
+            WriteRowsCsv(transactions, builder);
+			
+			var csvContent = builder.ToString();
+
+            return Encoding.UTF8.GetBytes(csvContent);
+        }
+
+        private static void WriteHeaderCsv(StringBuilder builder)
+        {
+            builder.AppendFormat("Transaction date,");
```

Comment:
What is the benefit of calling AppendFormat(...) (here and in method below) over calling simple Append()? There are no args so the more complex AppendFormat(...) call path isn't needed.

Reply:
I didn't write this code as it was pulled in from David's branch so I'll ask him and change accordingly. Its showing up in mine as its currently not in master

Reply:
Actioned